FrontPage/auth/AuthViewController.swift

```diff
@@ -212,4 +212,12 @@ extension AuthViewController {
     
   }
   
+  func getToken() -> String! {
+    return AuthViewController.self.token!
+  }
+  
+  func saveToken(token: String!){
+    AuthViewController.self.token = token
+  }
```

Comment:
these methods won't be necessary if we let the `AuthViewController` to set the token on the `Client`